js/build.js

```diff
@@ -1,4 +1,6 @@
-(function () {
+Fliplet.Widget.instance('list', function (data) {
+  var $container = $(this);
+
   $('.linked[data-list-item-id]').click(function (event) {
```

Comment:
The event listener attachment should be scoped to each widget instance.

```suggestion
  $container.find('.linked[data-list-item-id]').click(function (event) {
```

Reply:
@tonytlwu  $('.linked[data-list-item-id]') and $container.find('.linked[data-list-item-id]') equally return the array of elements with selector  .linked[data-list-item-id] and assign the event listener for them

Reply:
With `Fliplet.Widget.instance(name, fn)`, the `fn` function is called for every single instance of the component. If you have multiple instances of the component on the screen — try dropping in multiple instances on the same page — the `fn` function will be called multiple times (once for each instance).

If you don't scope the event listener to each instance, the event listener will be attached multiple times for each instance.